packages/angular_devkit/benchmark/src/command.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+export class Command {
```

Comment:
strikes me as overengineered but I'm pretty novice at this codebase and our design patterns...

Reply:
Simplified by removing the getters. They were there so the command did not change midway but realistically that's not going to happen.